src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -1251,7 +1252,7 @@ void AbstractNotationPaintView::setNotation(INotationPtr notation)
 
 void AbstractNotationPaintView::setReadonly(bool readonly)
 {
-    m_inputController->setReadonly(readonly);
+    m_readonly = readonly;
```

Comment:
Of course, we could add something here like 
```cpp
if (m_inputController)
    m_inputController->setReadonly(...)
```
But I assume you have checked that that is currently not necessary, right?

Reply:
added just in case something changes